src/pint/observatory/__init__.py

```diff
@@ -143,24 +144,6 @@ class Observatory:
     # standard name.
     _alias_map = {}
 
-    def __new__(cls, name, *args, **kwargs):
-        # Generates a new Observatory object instance, and adds it
-        # it the registry, using name as the key.  Name must be unique,
-        # a new instance with a given name will over-write the existing
-        # one only if overwrite=True
-        obs = super().__new__(cls)
-        if name.lower() in cls._registry:
-            if "overwrite" not in kwargs or not kwargs["overwrite"]:
-                raise ValueError(
-                    f"Observatory {name.lower()} already present and overwrite=False"
-                )
-            log.warning(f"Observatory '{name.lower()}' already present; overwriting...")
-
-            cls._register(obs, name)
-            return obs
-        cls._register(obs, name)
-        return obs
-
```

Comment:
Everything here can be done in `__init__()`. Having a non-trivial `__new__()` makes the objects non-copyable. That's why I am moving this code to `__init__()`.